### PR TITLE
1389 Update Watchlist to V2

### PR DESCRIPTION
**Pull request by SL-SWoods**
Update watch list design to match the V2 specs from UX. 
image
Changes
- Redesign of cards
- Pop over menu for remove/news/chart
- Removed drag out delete functionality. 
- Drag out only shows chart 
- Highlight of active buttons/charts etc
- Formatting change for preview window
- Thinner scroll bar
- Removed scrollbar arrows

### CODE REVIEW

**Comment by SL-SWoods** [comment]
Example of dragout window. 
image

File: packages/stockflux-watchlist/src/components/watchlist-card/WatchlistCard.css [hunk 1]
```diff
@@ -81,48 +36,110 @@
...
-  color: var(--icon-active);
+.card-menu .open {
+  border:solid 1px white;
+  background-color: #091A29;
 }
 
-.details-container {
-  width: 61%;
+.card-menu button:hover {
+  background-color: #091A29;
```
Comment (Johnston97) [inline packages/stockflux-watchlist/src/components/watchlist-card/WatchlistCard.css]: Could these be stored in a variable in app.css?

File: packages/stockflux-watchlist/src/components/watchlist-card/WatchlistCard.css [hunk 2]
```diff
@@ -81,48 +36,110 @@
...
+.card-menu > span {  
+  display: flex;
 }
 
-.icon {
-  align-self: baseline;
+.card-menu  button {
+  height:22px;  
+  border:transparent 1px solid;  
+  background-color: #1B3B56;
```
Comment (Johnston97) [inline packages/stockflux-watchlist/src/components/watchlist-card/WatchlistCard.css]: Same as above

**Comment by sarahlallemandsl** [comment]
Based on Friday's meeting I have written some notes with images, corrections and things to check over. 
Watchlist changes

**Pull request closed without merging**